lib/react-native-loader.js

```diff
@@ -8,9 +7,21 @@ util.querystring = require('querystring/');
 util.environment = 'js-react-native';
 
 var AWS = require('./core');
+module.exports = AWS;
+
+// // react-native specific modules
+ AWS.util.crypto.lib = util.crypto.lib;
```

Comment:
I was actually able to just import `lib/core.js` on the first line and mutate `util` directly on that, instead of pulling in `util`.